libraries/AP_Notify/Display_Backend.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "Display.h"
+
+#if  CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_BBBMINI
+#define OLED_I2C_BUS 2
+#else
+#define OLED_I2C_BUS 1
+#endif
```

Comment:
ok, follow-up commit added in which the bus and address are used in the frontend and the device is pass into the constructor.